data/contracts_finder/processing/scripts/cf_notices_ocds_api.py

```diff
@@ -0,0 +1,416 @@
+# coding=utf-8
+"""
+Gets OCDS JSON notices from the Contracts Finder API, and insert 100 with sufficient content,
+with an option to update these using the existing OCIDs
+
+API Info
+
+https://content-api.publishing.service.gov.uk/#gov-uk-content-api
+https://www.gov.uk/government/publications/open-contracting
+https://www.contractsfinder.service.gov.uk/apidocumentation/home
+https://www.contractsfinder.service.gov.uk/Published/Notice/releases/450280ac-b308-4df1-bdc5-9ede0c5c7e9f.json
+https://www.contractsfinder.service.gov.uk/Published/Notice/releases/5fecfc86-007a-4966-94e2-c695018ec2dd.json
+"""
+
+# from __future__ import print_function
+
+import argparse
+import copy
+from urllib3.util import Retry
+
+from requests.adapters import HTTPAdapter
+import json
+from datetime import datetime, timedelta, date
+from collections import OrderedDict
+from django.conf import settings
+import pandas as pd
+import logging
+import random
+import os
+
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search
+
+from ocdskit.upgrade import upgrade_10_11
+from ocdskit.combine import merge
+
+import requests
+import logging.config
+# logging.config.dictConfig({
+#     'version': 1,
+#     'disable_existing_loggers': True,
+# })
+
+logger = logging.getLogger(__name__)
+# logging.basicConfig(level=logging.INFO)
+
+ELASTICSEARCH_URL = os.getenv("ELASTICSEARCH_URL")
+DATABASE_URL = os.environ.get('DATABASE_URL')
+OPENOPPS_DB_URL = os.environ.get('OPENOPPS_DB_URL')
+OCDS_OUTPUT_DIR = os.path.join(settings.BASE_DIR, "data", "contracts_finder", "ocds")
+PROCESSING_DIR = os.path.join(settings.BASE_DIR, "data", "contracts_finder", "processing")
+supplier_df = pd.read_csv(os.path.join(settings.BASE_DIR, "data", "contracts_finder", "processing", 'openopps_supplier_matches.csv'), index_col='legal_name')
```

Comment:
global variables at the top of a file are usually just strings/settings, as if an error happened here running any functions it prevents the rest of the script loading for debugging